src/azure-cli/azure/cli/command_modules/sql/tests/latest/recordings/test_sql_server_ms_support_mgmt.yaml

```diff
@@ -2810,7 +2810,7 @@ interactions:
       - AZURECLI/2.46.0 (AAZ) azsdk-python-core/1.24.0 Python/3.10.10 (Linux-5.15.0-1033-azure-x86_64-with-glibc2.31)
         VSTS_7b238909-6802-4b65-b90d-184bca47f458_build_220_0
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.EventHub/namespaces/cliehnamespacems02?api-version=2022-01-01-preview
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.EventHub/namespaces/cliehnamespacems02?api-version=2022-10-01-preview
```

Comment:
The namespace commands are still using the old api. You should not replace it here. https://github.com/Azure/azure-cli/blob/d429c2f18abddabc720bcb05d3aedaadd53f0b7d/src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/namespace/_create.py#L27-L32